raiden/network/proxies/token_network_registry.py

```diff
@@ -228,3 +228,11 @@ def settlement_timeout_min(self) -> int:
     def settlement_timeout_max(self) -> int:
         """ Returns the maximal settlement timeout for the token network registry. """
         return self.proxy.contract.functions.settlement_timeout_max().call()
+
+    def get_token_network_created(self, to_block: BlockSpecification = "latest") -> int:
+        """ Returns the number of TokenNetwork contracts created so far in them
```

Comment:
perhaps you mean `in the` instead of `in them`

Reply:
Yes.